vault/identity_store.go

```diff
@@ -112,6 +112,11 @@ func (i *IdentityStore) paths() []*framework.Path {
 }
 
 func (i *IdentityStore) initialize(ctx context.Context, req *logical.InitializationRequest) error {
+	// Only primary should write the status
+	if i.core.ReplicationState().HasState(consts.ReplicationPerformanceSecondary | consts.ReplicationDRSecondary) {
```

Comment:
Needs to check for perf standby status, which I think may not be as simply as checking the const value here...if I remember correctly it doesn't show up in there for some reason. BK will know.

Reply:
Instead of using core in the backend, I am now using system view which all the other backends are relying on to determine the perf standby status.